packages/mdc-layout-grid/README.md

```diff
@@ -41,14 +41,12 @@ npm install --save @material/layout-grid
 
 ## The layout grid
 
-A grid consists of a group of cells, which get positioned in sequence according to a predefined number of columns.
-Cells specify how many columns to span (the default being 4), and get placed side by side while there is room. When
-there isn't enough room for a cell, it gets moved to the beginning of the next row, and placement continues as usual.
+A grid is a container that consists of a group of cells. The grids can define its own max-width or designate its column to be a certain width. Cells get positioned in sequence according to a predefined number of columns.
 
 The grid has 12 columns in desktop mode (>= 840px), 8 columns in tablet mode (>= 480px), and 4 columns in phone mode
-(< 480px). Column widths are variable; margins and gutters are fixed, with columns taking up the remainder of the space.
+(< 480px). Cells specify how many columns to span (the default being 4), and get placed side by side while there is room. When there isn't enough room for a cell, it gets moved to the beginning of the next row, and placement continues as usual.
```

Comment:
Cells specify how many columns to span (the default is 4). Cells are placed side by side until there is no more room, then the next cell is placed at the beginning of the next row.